nfts/frontend/src/utils/transferNft.js

```diff
@@ -0,0 +1,33 @@
+import fetch from "node-fetch";
+export default async function TransferNFTService(tokenId, toEmail, fromEmail) {
```

Comment:
Any reason why it is named as a service?

Reply:
fixed this